suites/suites/suites.yaml

```diff
@@ -381,7 +389,18 @@ handler_configurations:
       username: '{{system_tests_softlayer_username}}'
       api_key: '{{system_tests_softlayer_api_key}}'
 
-#####################################################################################
+  ################################################################################
+
+  aws_ec2:
+    inputs: inputs-aws-ec2.yaml
+    handler: ec2_handler
+    external: *aws_ec2_plugin_external
+    manager_blueprint: aws-ec2/aws-ec2-manager-blueprint.yaml
+    inputs_override:
+      aws_access_key_id: '{{system_tests_aws_access_key_id}}'
+      aws_secret_access_key: '{{system_tests_aws_secret_access_key}}'
```

Comment:
aren't you missing the `image_id` and `instance_type` inputs?
In order to avoid duplicating these here and in the properties, look at the `variables` section on top, and add these with reasonable names there, then in the properties and inputs sections you can use `'{{my_var_name'}}` instead of hard coded `ami-3cf8b154` (for example)